### DISCUSSION TITLE
Many Directory creations

### DISCUSSION THREAD

**Discussion by Jazz9** [minio/minio#17117] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/39378577/235903236-d850b090-aae5-4038-890e-da65d238fd14.png)

This is on a 12 drive x 6 servers cluster running Veeam Backups. Veeam seem to create many folders and then Minio create some additional ones.

I wonder if the excessive folder writes are causing performance issues? Is this common to see?

I can't really see how this can be improved.

Glen

**Comment by klauspost** [comment, marked as answer]
This is how Veeam is designed. They may modify their placement strategy in the future.
